ampersand-filter-tracker.js

```diff
@@ -0,0 +1,140 @@
+;(function() {
+  var d3 = require('d3');
+  var _ = require('lodash');
+
+  var AmpersandState = require('ampersand-state');
+  var AmpersandView = require('ampersand-view');
+
+  var FilterTrackerState = AmpersandState.extend({
+    props: {
+      filter: 'function'
+    },
+    session: {
+      handles: 'array'
+    },
+    initialize: function() {
+      _.each(this.handles, function(handle) {
+        console.log(handle.model);
+        handle.active = false;
+        _.each(handle.props, function(prop) {
+          handle.model.on('change:' + prop, function(model, prop) {
+            handle.active = false;
+            _.each(handle.props, function(prop, index) {
+              if ((typeof handle.clearValues[index] !== 'function' && model[prop] !== handle.clearValues[index]) || (typeof handle.clearValues[index] === 'function' && handle.clearValues[index].call(handle.model))) {
+                handle.active = true;
+              }
+            });
+            this._view.renderFilters(this.handles);
+            this.filter = this.generateFilter(this.handles);
+          }.bind(this));
```

Comment:
when using `_` methods, you can typically avoid needing to bind and can just pass `this` as the context.  May be simpler - either's fine by me